File: gui2/src/domain/stub/full-stub-overview-model.ts
import type { StubOverviewModel } from "@/domain/stub/stub-overview-model";
import type { StubMetadataModel } from "@/domain/stub/stub-metadata-model";

export interface FullStubOverviewModel {
  stub: StubOverviewModel;
  metadata: StubMetadataModel;
}

File: src/HttPlaceholder.Domain/FullStubOverviewModel.cs
namespace HttPlaceholder.Domain;

/// <summary>
///     A class for storing a stripped down version of a stub with metadata.
/// </summary>
public class FullStubOverviewModel
{
    /// <summary>
    ///     Gets or sets the stub.
    /// </summary>
    public StubOverviewModel Stub { get; set; }

    /// <summary>
    ///     Gets or sets the metadata.
    /// </summary>
    public StubMetadataModel Metadata { get; set; }
}
